src/services/facilitate.service.js

```diff
@@ -0,0 +1,44 @@
+import Axios from 'axios';
+
+export default class FacilitateService {
+  static serviceInstance = null;
+
+  constructor() {
+    this.baseURL = '/facilitates';
+    this._setAuthHeaders();
+  }
+
+  static getInstance() {
+    if (FacilitateService.serviceInstance == null) {
+      FacilitateService.serviceInstance = new FacilitateService();
+    }
+
+    return this.serviceInstance;
+  }
+
+  _setAuthHeaders() {
```

Comment:
i was able to submit without being logged in - is this expected?

Reply:
yes because the idea is anyone using the app (including people without credentials) can post a request to add a facility